src/stories/TransactionPreviewContent.stories.tsx

```diff
@@ -0,0 +1,264 @@
+    containerStyles: [{margin: 10}],
+};
+
+const disabledProperties = Object.keys(staticProps).reduce<Record<string, {table: {disable: boolean}}>>((storyProperties, fakeDataKey) => {
+    // eslint-disable-next-line no-param-reassign
+    storyProperties[fakeDataKey] = {
+        table: {
+            disable: true,
+        },
+    };
+    return storyProperties;
+}, {});
+
+type TransactionPreviewStory = StoryFn<typeof TransactionPreviewContent>;
+
+const usersArgType: InputType = {
+    options: availableUsers,
+    mapping: usersMap,
+    control: 'select',
+    if: {
+        arg: 'isIOU',
+    },
+};
+
+/**
+ * We use the Component Story Format for writing stories. Follow the docs here:
+ *
+ * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
+ */
+const story: Meta<typeof TransactionPreviewContent> = {
+    title: 'Components/TransactionPreview',
+    component: TransactionPreviewContent,
+    argTypes: {
+        ...disabledProperties,
+        requestCurrency: {
+            control: 'select',
+            options: ['USD', 'PLN', 'EUR'],
+            if: {
+                arg: 'isBillSplit',
+            },
+        },
+        splitShare: {
+            control: 'number',
+            if: {
+                arg: 'isBillSplit',
+            },
+        },
+        RBRmessage: {
+            control: 'text',
+            if: {
+                arg: 'shouldShowRBR',
+            },
+        },
+        receiptImages: {
+            mapping: receiptImagesMap,
+            options: availableColors,
+            control: 'select',
+        },
+        from: usersArgType,
+        to: usersArgType,
+    },
+};
+
+function Template(props: TransactionPreviewUIProps) {
+    return (
+        <ThemeProvider theme={CONST.THEME.LIGHT}>
+            <ThemeStylesProvider>
+                <View style={{flexDirection: 'row'}}>
+                    {/* eslint-disable react/jsx-props-no-spreading */}
+                    <TransactionPreviewContent
+                        {...staticProps}
+                        {...props}
+                    />
+                    {/* eslint-enable react/jsx-props-no-spreading */}
+                </View>
+            </ThemeStylesProvider>
+        </ThemeProvider>
+    );
+}
+
+// Arguments can be passed to the component by binding
+// See: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Default: TransactionPreviewStory = Template.bind({});
+const SplitAndKeepButton: TransactionPreviewStory = Template.bind({});
+const MoreThan1ImageAndRBR: TransactionPreviewStory = Template.bind({});
+const MoreThan1ImageAndSplit: TransactionPreviewStory = Template.bind({});
+const DeletedSplitWithRBRAndKeepButton: TransactionPreviewStory = Template.bind({});
+const IOUWithKeepButton: TransactionPreviewStory = Template.bind({});
+
+Default.args = {
+    ...generateProps('$60.00', 'New Jersey Office', 'Acme', 'Nov 19', 'blue', 'Grocery stores', 'Cash'),
```

Comment:
it seems to me that we can draw common arguments to args in:
`const story: Meta<typeof TransactionPreviewContent> = {`
, as we said offline